src/TopApp.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import {
+  BrowserRouter as Router,
+  Switch,
+  Route,
+} from 'react-router-dom';
+import axios from 'axios';
+import { Container } from 'semantic-ui-react';
+import { ToastContainer } from 'react-toastify';
+import App from './App';
+// import PrivateRoute from './PrivateRoute';
+import { UserContext, userContext } from './userContext';
+import 'react-toastify/dist/ReactToastify.css';
+
+axios.defaults.baseURL = 'http://localhost:4000/api';
+
+export default function TopApp() {
+  // Example of setting isAuthenticated in userContext, This function is for demonstration.
+  function handleOnAuthenticated(username) {
+    userContext.isAuthenticated = true;
+    userContext.username = username;
```

Comment:
Need clarification how it should work. Could You add some manual steps how to log in? If it just exists in the project.

Reply:
I will try to add some manual steps otherwise we will get rid of the function. Will be added whoever is implementing authentication.

Reply:
Are you sure that the old way to declare context in app is a good solution? What about use hooks ? `to discuss`